src/staff/views/StaffList/StaffList.tsx

```diff
@@ -145,90 +134,94 @@ export const StaffList: React.FC<StaffListProps> = ({ params }) => {
     variables: DEFAULT_INITIAL_SEARCH_DATA
   });
 
+  const [staffMemberAdd, staffMemberAddOpts] = useStaffMemberAddMutation({
+    onCompleted: data => {
+      if (data.staffCreate.errors.length === 0) {
+        notify({
+          status: "success",
+          text: intl.formatMessage(commonMessages.savedChanges)
+        });
+        navigate(staffMemberDetailsUrl(data.staffCreate.user.id));
+      }
+    }
+  });
+
+  const handleStaffMemberAdd = async (formData: AddMemberFormData) => {
```

Comment:
Just a thought, but could look nicer with a spread 

``` 
async ({permissionGroups, email, firstName, lastName }: AddMemberFormData) => {
```

```
  addGroups: permissionGroups,
  email,
  firstName,
  lastName,

Reply:
Don't want to fix all the things along the way in this PR, just migrate to hooks. This way it won't take an eternity and won't produce too big PRs.